contents/performance/WebPerformance.md

```diff
@@ -0,0 +1,79 @@
+- Sites which take more than 3 seconds to load are [more likely to be abandoned by users](https://developer.akamai.com/blog/2016/09/14/mobile-load-time-user-abandonment).
+- Conversely, reducing page load times have been shown to have significant impact on [improving user engagement](https://medium.com/carousell-insider/how-we-made-carousells-mobile-web-experience-3x-faster-bbb3be93e006).
+- In addition, search engines have begun to use page load times as a [factor in determining search rankings](https://webmasters.googleblog.com/2018/01/using-page-speed-in-mobile-search.html).
+- Improving web performance is also essential in making your web application [accessible to users from emerging markets](https://building.calibreapp.com/beyond-the-bubble-real-world-performance-9c991dcd5342), where low-end devices and limited bandwidth are the norm.
+
+In order to deliver a positive user experience, web developers must ensure that their applications meet acceptable performance standards.
+
+## Key Ideas in Improving Web Performance
+There are many factors which affect web performance. Here, we give an overview of some of the key ideas that can be used to improve the performance of your web application.
+
+**Reduce Javascript Payloads**
+
+Loading and executing Javascript is often [the slowest part](https://medium.com/@addyosmani/the-cost-of-javascript-in-2018-7d8950fbb5d4) of the page load process. Reducing the amount of Javascript that needs to be loaded thus significantly reduces the time taken for your site to load.
+
+Here are some steps that web developers can take to reduce the amount of Javascript that clients need to load:
+- Use [code splitting](https://developers.google.com/web/fundamentals/performance/optimizing-javascript/code-splitting/) to load only the Javascript required for the page being accessed.
+- Remove unused code (often parts of libraries that are not used in the application) with [tree shaking](https://developers.google.com/web/fundamentals/performance/optimizing-javascript/tree-shaking/).
+- Ensure that the delivered Javascript is [minified](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/javascript-startup-optimization/).
+- Create [performance budgets](https://infrequently.org/2017/10/can-you-afford-it-real-world-web-performance-budgets/) that specify the maximum size of Javascript payloads,and [enforce them](https://web.dev/fast/incorporate-performance-budgets-into-your-build-tools) to ensure that the size of your Javascript payloads remain under control.
+
+**Optimize Images**
+
+Images form a [significant portion](https://httparchive.org/reports/page-weight) of the resources loaded on web applications. A significant amount of time and bandwidth is used to load images, making the site feel slower to users. To improve performance, Web developers should thus try to reduce the amount of bandwidth used to load images.
```

Comment:
Feels like this can be more concise. Maybe replace this:
> A significant amount of time and bandwidth is used to load images, making the site feel slower to users. To improve performance, Web developers should thus try to reduce the amount of bandwidth used to load images.

With this?
> To improve performance and speed up the site for users, web developers should thus try to reduce the amount of bandwidth used to load images.